spec/factories/laundries.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :laundry do
```

Comment:
Hmm... not sure you should settle for a string for schedule

Reply:
done